crates/wasmtime/src/func/typed.rs

```diff
@@ -321,7 +321,7 @@ unsafe impl WasmTy for Option<ExternRef> {
 
```

Comment:
Where does ExternRef come from here?

Reply:
It is defined in https://github.com/bytecodealliance/wasmtime/blob/main/crates/wasmtime/src/ref.rs#L10